visualizer/examples.js

```diff
@@ -162,6 +226,12 @@
         setExample(selectedId, cm.getValue());
         saveExamples();
       }
+    },
+    'Alt-S': function(cm) { // save (windows)
```

Comment:
Yep, normally you just need to call `e.preventDefault()` on the appropriate event.

However, I believe that CodeMirror's "extraKeys" functionality already handles this, because Cmd-S is save on Mac OS, and the existing Cmd-S stuff that's here doesn't trigger a browser save dialog for me.